File: Python/_191_Number_of_1_Bits/solution.py
class Solution(object):
    def hammingWeight(self, n):
        """
        :type n: int
        :rtype: int
        """
        result = 0
        for i in range(0, 32):
            if n & 1 > 0:
                result += 1
            n >>= 1
        return result

File: Java/_191_Number_of_1_Bits/Solution.java
public class Solution {
    // you need to treat n as an unsigned value
    public int hammingWeight(int n) {
        int result = 0;
        for (int i = 0; i < 32; i++, n >>= 1)
            if ((n & 1) > 0)
                result++;
        return result;
    }
}